app/controllers/application_controller.rb

```diff
@@ -1,2 +1,9 @@
 class ApplicationController < ActionController::Base
+  helper_method :current_user
+
+  private
+
+  def current_user
+    @current_user ||= User.find(session[:user_id]) if session[:user_id]
+  end
```

Comment:
[OPTIONAL]
- [ ] Nice work implementing the `current_user` method in the `applicationController` class :clap: However, according to the project requirements the `current_user` method should return the first user from the database. Therefore I suggest that you update the the method to return the first user from the database to follow the project requirement.

Example:
```rb
...

def current_user
  User.first
end
```
_This is a requirement of the project but the first reviewer didn't notice it. so I requested this as an optional but I highly recommend you to make the change_